.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:jammy
+FROM ubuntu:latest
```

Comment:
I've made it point to `latest` as this implies a few changes to the docker backbone for what concerns X11. I did it for not losing these important changes.

Being packages still pointing to `jammy` instead, I wouldn't expect them to work flawlessly.